selfdrive/car/volkswagen/carstate.py

```diff
@@ -253,8 +248,17 @@ def update_pq(self, pt_cp, cam_cp, ext_cp, trans_type):
     # Additional safety checks performed in CarInterface.
     ret.espDisabled = bool(pt_cp.vl["Bremse_1"]["ESP_Passiv_getastet"])
 
+    self.frame += 1
     return ret
 
+  def update_hca_state(self, hca_status):
+    # Treat INITIALIZING and FAULT as temporary for worst likely EPS recovery time, for cars without factory Lane Assist
+    # DISABLED means the EPS hasn't been configured to support Lane Assist
+    self.eps_init_complete = self.eps_init_complete or (hca_status in ("DISABLED", "READY", "ACTIVE") or self.frame > 600)
```

Comment:
In place of `self.frame > 600`, what I really want is CarController startup + 2 seconds, but not sure how to do that. I could pass `canValid` down into `CS.update()` but that's a crude proxy and would need latching. Suggestions welcome.